packages/react-monaco-editor/scripts/copyTypes.ts

```diff
@@ -8,14 +8,14 @@ import * as path from 'path';
 export function copyTypes() {
   const packagesToResolve = ['@fluentui/react', '@fluentui/react-hooks', '@fluentui/example-data'];
   const resolvedPackages = [];
-  const pathsToCopy = [];
+  const pathsToCopy: string[] = [];
 
   while (packagesToResolve.length) {
-    const pkg = packagesToResolve.shift();
+    const pkg = packagesToResolve.shift() as string;
```

Comment:
why is this cast needed? typescript should know that it's a string from line 9

Reply:
shift() returns `T` or `undefined` if array is empty, which in our case will never happen. but with existing construct TS cannot understand that that situation never happens, thus the assertion (which will not throw runtime error)